config/locales/en.yml

```diff
@@ -9,9 +9,51 @@ en:
         client_group: 'Client group'
         contact_information: 'Contact information'
         links_to_research: 'Links to further information'
+      evidence_summary_search_form:
+        keyword: 'keyword'
+        years_of_publication:
+          - All years
+          - Last 2 years
+          - Last 5 years
+          - More than 5 years ago
+        client_groups:
+          - Children (3 - 11)
+          - Young people (12 - 16)
+          - Parents/families
+          - Young adults (17 - 24)
+          - Working age (18 - 65)
+          - Older people (65+)
+          - Over-indebted people
+          - Social housing tenants
+          - Teachers/practitioners
+          - Other
+        topics:
+          - Pensions
+          - Pensions and retirement planning
+          - Credit use and debt
+          - Budgeting and keeping track
+          - Insurance and protection
+          - Financial education
+          - Financial capability
+        countries_of_delivery:
+          - United Kingdom
+          - England
+          - Northern Ireland
+          - Scotland
+          - Wales
+          - USA
+          - Other
   evidence_hub:
     summaries:
       header: 'Evidence Summaries'
+    filters_header: 'Filter your search'
+    filters:
+      checkbox:
+        - client_groups
```

Comment:
are these translations? Perhaps I am not understanding, but these strike me as constants that should be within the model?